app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -97,27 +97,34 @@ export function SettingsScreen ({ navigation }: Props): JSX.Element {
         text={translate('screens/Settings', 'SECURITY')}
       />
 
-      <SecurityRow
+      <NavigateItemRow
         label='Recovery Words'
         onPress={revealRecoveryWords}
         testID='view_recovery_words'
       />
 
       {
         isEncrypted && (
-          <SecurityRow
+          <NavigateItemRow
             label='Change Passcode'
             onPress={changePasscode}
             testID='view_change_passcode'
           />
         )
       }
 
-      <RowNavigateItem
-        pageName='AboutScreen'
-        title='About'
+      <View style={tailwind('mt-4')} />
+      <NavigateItemRow
+        testID='setting_navigate_About'
+        label='About'
+        onPress={() => navigation.navigate('AboutScreen')}
       />
       <RowThemeItem />
+      <NavigateItemRow
+        testID='setting_navigate_language_selection'
+        label='Language'
```

Comment:
This is not translated

Reply:
label prop is translated within `NavigateItemRow`